examples/third_party/python/BUILD.python3.bazel

```diff
@@ -15,7 +15,7 @@ configure_make(
     }),
     configure_options = [
         "--with-openssl=$EXT_BUILD_DEPS/openssl",
-        "--with-zlib=$EXT_BUILD_DEPS/zliasdfb",
+        "--with-zlib=$EXT_BUILD_DEPS/zlib",
```

Comment:
I think in this case bazel would just fetch the cached build. Perhaps re-ordering this list would trigger a new build?